### ISSUE TITLE
Nice URLs enabled for create user account by default

### ISSUE THREAD

**Issue by billsargent** [shish/shimmie2#712]
**Server Software**
(You can get all these stats from `http://<your site>/system_info`)

  shimmie: "2.8.2+"
  schema: 18
  php: "7.3.11-0ubuntu0.19.10.3"
  db: "sqlite 3.29.0"
  os: "Linux ubuntu-desktop 5.3.0-42-generic #34-Ubuntu SMP Fri Feb 28 05:49:40 UTC 2020 x86_64"
  server: "Apache/2.4.41 (Ubuntu)"

**Client Software (please complete the following information)**
 - Device: [e.g. iphone, windows desktop]
 - Browser: [e.g. chrome, safari]

**What steps trigger this bug**
1: Click create new account with a fresh install
2: Fill in the details of the first account (admin) and press the create button.
**What did you expect to happen?**
For it to create the account.

**What actually happened?**

Nice URLs were not enabled, nor htaccess. I expected by default nice urls to be disabled in Shimmie2 but for this specific action, it tried to use a nice url to submit the create user form resulting in a 404 Error. 

Enabling htaccess solved the problem, however, nice urls should and is disabled by default, so it should not have tried to use it there. It didn't do that in the past versions.

**Comment by shish** [comment]
Ack, thanks for the report, 7b9c9dc2 should fix that :)

**Comment by billsargent** [comment]
ooh my first ever bug report :D Thanks. It had been awhile since I had set up shimmie2 and apache together. All of this is just a pet project for me to kill time indoors. I've scraped most of rule34 paheal net and i've got the data and just wanted to see how importing that much data would work. I've been working on this off and on for a few years. I just get sidetracked. But now there's time at home... alone.... :) Thanks for fixing it!
